apps/web/features/banner/banner.tsx

```diff
@@ -18,32 +17,33 @@ import {
 } from '@shared/ui';
 
 interface BannerProps {
-  type: 'desktop' | 'mobile';
-  initialVisibility: boolean;
   banner: TypeFromSelection<typeof bannerSelection>;
 }
 
-export default function Banner({
-  type,
-  banner,
-  initialVisibility,
-}: BannerProps) {
-  const [isVisible, setIsVisible] = useState(initialVisibility);
-  const pathname = usePathname();
-  const isSpammening = pathname === '/spammening';
+export default function Banner({ banner }: BannerProps) {
+  const [isBannerClosed, setIsBannerClosed] = useState(true);
 
-  const handleClose = () => {
-    setIsVisible(false);
-    document.cookie =
-      'polkadot_banner_closed=true; path=/; max-age=259200; samesite=strict; secure';
-  };
+  useEffect(() => {
+    function checkBannerCookie() {
+      return document.cookie
+        .split('; ')
+        .some((cookie) => cookie.startsWith('polkadot_banner_closed=true'));
```

Comment:
I don't think that this is correct. You can't be sure that the whole cookie string will start with this specific cookie. I think the right approach would be to get this specific cookie with:

```typescript
const cookie = await window.cookieStore.get('polkadot_banner_closed');
```